docs/foundations/typography/font-family/index.md

```diff
@@ -0,0 +1,124 @@
+The Red Hat font family includes 3 fonts: Red Hat Display, Text, and Mono. Their design is based on who we are: real people with big ideas creating platforms that power technology innovation.
+
+To learn more about the Red Hat font family, go to the [Brand standards][brandstandards] website.
+
+## Design tokens
+
+Design tokens represent our design decisions as code. They are also how we track changes made to the design system. If a text style is changed, that change propagates across all websites and experiences that use tokens.
+
+To see a list of tokens, including Noto, go to the [Typography tokens][typographytokens] page.
+
+## Red Hat Display
+
+Red Hat Display is our default font. Use it for headings to grab attention and establish hierarchy. Its letterforms reinforce our [tone of voice][tone] and reflect our technical and engineering expertise. Meanwhile, human touches disrupt the rigidity to represent our people and call back to the history of our brand.
+
+Use regular, medium, or bold weights. Other weights can be used for [Expressive type][expressivetype] only.
+
+<uxdot-example width-adjustment="810px" color-palette="lightest">
+  <img src="./typography-rh-display.svg"
+      alt="examples of Red Hat Display and its three weights"
+      width="810"
+      height="192">
+</uxdot-example>
+
+## Red Hat Text
+
+Red Hat Text takes all of the personality from Display and optimizes it for more demanding applications like paragraph text or text at small sizes.
+
+Use the medium weight and italics for emphasis. Other weights can be used for [Expressive type][expressivetype] only.
+
+<uxdot-example width-adjustment="810px" color-palette="lightest">
+  <img src="./typography-rh-text.svg"
+      alt="examples of Red Hat Text and its weights and styles"
+      width="810"
+      height="224">
+</uxdot-example>
+
+## Red Hat Mono
+
+Red Hat Mono was created to distinguish code from natural-language text. Use it to demonstrate code snippets in communications and interfaces or as a stylistic approach for more technical audiences.
+
+Use the medium weight and italics for emphasis. Other weights can be used for [Expressive type][expressivetype] only.
+
+<uxdot-example width-adjustment="810px" color-palette="lightest">
+  <img src="./typography-rh-mono.svg"
+      alt="examples of Red Hat Mono and its weights and styles"
+      width="810"
+      height="192">
+</uxdot-example>
+
+## Non-Latin fonts
+
+Noto is a set of international fonts developed by Google, Monotype, Adobe, and leading experts from around the world. All Noto fonts are licensed under the Open Font License. The simple and clean design makes it a good counterpart to the Red Hat font family.
+
+<uxdot-example width-adjustment="778px" color-palette="lightest">
+  <img src="./typography-non-latin-fonts.svg"
+      alt="examples of Noto used for non-latin text"
+      width="778"
+      height="207">
+</uxdot-example>
+
+## Download fonts
+
+### Red Hat
+<div class="grid sm-three-columns">
```

Comment:
Just my opinion maybe, but I think Tiles might make the affordance nicer.

<img width="758" alt="image" src="https://github.com/user-attachments/assets/16b29fc7-85d4-457b-9736-71e38e9ae122" />

Reply:
I see what you mean about these working well if you could click anywhere within the card, but I think using a tile would just break the text and the arrow to different lines. I kept the cards for now, but we can update later if others feel the same.